lib/spack/spack/test/env.py

```diff
@@ -48,3 +51,58 @@ def test_activate_should_require_an_env():
 
     with pytest.raises(TypeError):
         ev.activate(env=None)
+
+
+packages_high = {
+    'packages': {
+        'foo': {
+            'version': ['a']
+        },
+        'bar': {
+            'version': ['b'],
+        },
+    }
+}
+
+
+def test_env_include_packages_file(tmpdir, mutable_empty_config):
```

Comment:
A couple things:

* Similar tests are in `spack/test/cmd/env.py` so I think this should go there (or the similar tests there should move here)
* I think there could be some consolidation of logic between this and `test_env_with_included_config_file`

Reply:
🤔